### ISSUE TITLE
Add-on echo command does not respect leading white spaces

### ISSUE THREAD

**Issue by jorgemoralespou** [minishift/minishift#1128]
I want to indent output of echo command in an addon:

I have:

````
echo You have installed #{addon-name}
echo To access #{addon-name} go to http://cockpit.#{routing-suffix}
echo or
echo Use minishift: 
echo     minishift openshift service cockpit -n cockpit
echo 
echo To delete:
echo     oc delete oauthclients/cockpit-oauth-client project/cockpit --as=system:admin
````

I get:

````
-- Applying addon 'cockpit':....
  You have installed cockpit
  To access cockpit go to http://cockpit.192.168.64.2.nip.io
  or
  Use minishift:
  minishift openshift service cockpit -n cockpit

  To delete:
  oc delete oauthclients/cockpit-oauth-client project/cockpit --as=system:admin
````

As can be seen indentation is removed

**Comment by hferentschik** [comment]
I think we might be trimming the string to echo. Should be easy to fix ...

**Comment by hferentschik** [comment]
Merged via pull request #1131
